auth/authn.go

```diff
@@ -46,14 +44,24 @@ var (
 
 	Authority *authority.Authority
 
-	authEnabled bool
-	signingKey  []byte
-	tokenStore  TokenStore
+	authEnabled      bool
+	signingKeyBase64 []byte
+	signingKeyBase32 string
```

Comment:
Why are you storing this as string instead of []byte? I think every time you use this variable you are converting it to []byte.